roles/mss/templates/operator.yml.j2

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mobile-security-service-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: mobile-security-service-operator
+  template:
+    metadata:
+      labels:
+        name: mobile-security-service-operator
+    spec:
+      serviceAccountName: mobile-security-service-operator
+      containers:
+        - name: mobile-security-service-operator
+          # Replace this with the built image name
+          image: {{ mobile_security_service_operator_image }}
```

Comment:
As we've just got the image tag here is it possible to import this file as we discussed yesterday?

Reply:
yes, it should be possible. The only blocker for now is the same file in the repo always references the master tag. see https://github.com/aerogear/mobile-security-service-operator/blob/0.4.1/deploy/operator.yaml#L19.

Once that is fixed, we can import the file instead.